packages/design-system-scripts/gulp/build.js

```diff
@@ -52,6 +53,45 @@ async function copyAll(dir) {
   return Promise.all(copyTasks);
 }
 
+/**
+ * Similar to compileJS but babel is configured for esmodules
+ */
+async function compileEsmJs(dir) {
+  const src = path.join(dir, 'src', 'components');
+
+  // Create esm entry file
+  await fs.writeFileSync(path.join(dir, 'dist', 'index.es.js'), `export * from './esnext';`);
```

Comment:
I created 2 entry points (1 for ESM, 1 for CommonJS) by moving all the `exports` into an `index.js` in the `components` folder, so that I could easily generate a new `index.es.js` entry point without worrying about changing all the import paths. 

AKA, I didn't have to worry about doing this change:
```
export * from './components/Alert/Alert';

export * from './esnext/Alert/Alert'
```